yaksh/management/commands/create_demo_course.py

```diff
@@ -0,0 +1,77 @@
+import os
+from django.core.management.base import BaseCommand
+from yaksh.models import Course, Question, Quiz, QuestionPaper, Profile, FileUpload
+from yaksh.file_utils import extract_files
+from django.contrib.auth.models import User
+from django.utils import timezone
+from django.core.files import File
+from datetime import datetime, timedelta
+import pytz
+import zipfile
+
+
+def create_demo_course():
+    """ creates a demo course, quiz """
+
+    success = False
+    print("Creating Demo User...")
+    # create a demo user
+    user, u_status = User.objects.get_or_create(username='demo_user',
+                                                password='demo',
+                                                email='demo@test.com')
+    Profile.objects.get_or_create(user=user, roll_number=0,
+                                  institute='demo_institute',
+                                  department='demo_department',
+                                  position='Faculty')
+
+    print("Creating Demo Course...")
+    # create a demo course
+    course, c_status = Course.objects.get_or_create(name="Demo_course",
+                                                    enrollment="open",
+                                                    creator=user)
+
+    print("Creating Demo Quiz...")
+    # create a demo quiz
+    quiz, q_status = Quiz.objects.get_or_create(start_date_time=timezone.now(),
```

Comment:
If I run the command again then again it will create a new quiz followed by new question paper, since timezone.now() will change each time.